scrapers.js

```diff
@@ -302,19 +320,35 @@ let scrapers = [
       let $ = await fetch.page(this.url);
 
       let cases = 0;
-      cases += parse.number($('p:contains("Number of PHL positives")').first().text().split(': ')[1]);
-      cases += parse.number($('p:contains("Number of commercial lab positives")').first().text().split(': ')[1]);
+      cases += parse.number(
+        $('p:contains("Number of PHL positives")')
+          .first()
+          .text()
+          .split(': ')[1]
```

Comment:
Chaining is much clear like this, so let's keep it the way it is in your PR.

Can we make prettier do this automatically before commits?

Reply:
on saves, yes. not sure on commits? I'll verify